src/app/components/RuntimeEvents/RuntimeEventDetails.tsx

```diff
@@ -464,6 +508,42 @@ const RuntimeEventDetailsInner: FC<{
           </StyledDescriptionList>
         </div>
       )
+    case RuntimeEventType.roflmarketprovider_created:
+    case RuntimeEventType.roflmarketprovider_updated:
+    case RuntimeEventType.roflmarketprovider_removed:
+      return (
+        <div>
+          <EventTypeIcon eventType={event.type} />
+          <StyledDescriptionList titleWidth={isMobile ? '100px' : '200px'}>
+            <MaybeEventErrorLine event={event} />
+            <dt>{t('common.address')}</dt>
+            <dd>
+              <AccountLink scope={scope} address={event.body.address} />
+            </dd>
+          </StyledDescriptionList>
+        </div>
+      )
+    case RuntimeEventType.roflmarketinstance_created:
+    case RuntimeEventType.roflmarketinstance_updated:
+    case RuntimeEventType.roflmarketinstance_accepted:
+    case RuntimeEventType.roflmarketinstance_cancelled:
+    case RuntimeEventType.roflmarketinstance_removed:
+    case RuntimeEventType.roflmarketinstance_command_queued:
+      return (
+        <div>
+          <EventTypeIcon eventType={event.type} />
+          <StyledDescriptionList titleWidth={isMobile ? '100px' : '200px'}>
+            <MaybeEventErrorLine event={event} />
+            <dt>{t('rofl.provider')}</dt>
+            <dd>
+              <AccountLink scope={scope} address={event.body.provider} />
+            </dd>
+            <dt>{t('rofl.instanceId')}</dt>
```

Comment:
This should be specific to roflmarket, not rofl, because it isn't related to t('rofl.activeInstances'). Both use the same word "instance" for very different meanings

Reply:
Should rename as https://github.com/oasisprotocol/cli/pull/422#discussion_r2046912838?